src/Schema/Tables/Medications/PatientMedications.sql

```diff
@@ -0,0 +1,28 @@
+﻿CREATE TABLE [dbo].[PatientMedications]
+(
+    [PatientMedicationsId] BIGINT NOT NULL IDENTITY(1, 1),
+    [PatientId]            INT NOT NULL,
+    [UserId]               INT NOT NULL,
+    [MedicationId]         BIGINT NOT NULL,
+    [AssignmentDate]       DATE NOT NULL DEFAULT Getdate(),
+    [Duration]             NVARCHAR(20) NOT NULL,
+    [Notes]                NVARCHAR(max) NULL,
+    -- Сurrent treatment by medicine equals 0, finished equals 1.
+    [Status]               BIT NOT NULL DEFAULT 0
```

Comment:
May be not Status, but something like Finished? Than you will not need that comment before column

Reply:
What about IsFinished